### PR TITLE
Move GPU CI to the new expansion

### PR DESCRIPTION
**Pull request by trontrytel**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -24,6 +22,7 @@ steps:
       - julia --project=test -e 'using CUDA; CUDA.precompile_runtime()'
       - julia --project=test -e 'using Pkg; Pkg.status()'
     agents:
+      partition: gpu
```
Comment (Sbozzolo) [inline .buildkite/pipeline.yml:25]: I think you can remove this if you want. It should pick up the gpu queue anyway.

**Review by Sbozzolo — COMMENTED** [review]
You also have to remove the [JuliaProject.toml](https://github.com/CliMA/CloudMicrophysics.jl/blob/main/.buildkite/JuliaProject.toml) file

**Review by Sbozzolo — APPROVED** [review]

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -1,15 +1,13 @@
 agents:
   queue: central
```
Comment (Sbozzolo) [inline .buildkite/pipeline.yml:2]:
```suggestion
  queue: new-central
```

I approved but we need this.

Comment (trontrytel) [reply .buildkite/pipeline.yml:2]: argh! Thank you

**Pull request merged**